documentation/src/docs/metrics.md

```diff
@@ -7,491 +7,466 @@ You may obtain a copy of the License at
-```
-segmentstore.bookkeeper.write_bytes
-segmentstore.bookkeeper.bookkeeper_ledger_count - with tag {"container", $containerId}
-```
-
-- Tier-2 Storage read/write latency metrics (histogram):	
-```
-segmentstore.storage.read_latency_ms
-segmentstore.storage.write_latency_ms
-```
-
-- Tier-2 Storage read/write data and file creation metrics (counters):
-```
-segmentstore.storage.read_bytes
-segmentstore.storage.write_bytes
-segmentstore.storage.create_count
-```
-
-- Segment Store container-specific operation metrics:
-```
-// Histograms - all with tags {"container", $containerId}
-segmentstore.container.process_operations.latency_ms
-segmentstore.container.process_operations.batch_size
-segmentstore.container.operation_queue.size
-segmentstore.container.operation_processor.in_flight
-segmentstore.container.operation_queue.wait_time
-segmentstore.container.operation_processor.delay_ms
-segmentstore.container.operation_commit.latency_ms
-segmentstore.container.operation.latency_ms
-segmentstore.container.operation_commit.metadata_txn_count
-segmentstore.container.operation_commit.memory_latency_ms
-
-// Gauge
-segmentstore.container.operation.log_size
-```
-
-- Segment Store operation processor (counter) metrics - all with tags {"container", $containerId}
-```
-// Counters/Meters
-segmentstore.container.append_count
-segmentstore.container.append_offset_count
-segmentstore.container.update_attributes_count
-segmentstore.container.get_attributes_count
-segmentstore.container.read_count
-segmentstore.container.get_info_count
-segmentstore.container.create_segment_count
-segmentstore.container.delete_segment_count
-segmentstore.container.merge_segment_count
-segmentstore.container.seal_count
-segmentstore.container.truncate_count
-
-```
-
-- Segment Store active Segments (gauge) and thread pool status (histogram) metrics:
-```
-// Gauge - with tags {"container", $containerId}
-segmentstore.active_segments
-
-// Histograms
-segmentstore.thread_pool.queue_size
-segmentstore.thread_pool.active_threads
-```
+- Segment Store Read/Write latency of storage operations ([Histograms](https://micrometer.io/docs/concepts#_histograms_and_percentiles)):
+
+   ```
+    pravega_segmentstore.segment.create_latency_ms
+    pravega_segmentstore.segment.read_latency_ms
+    pravega_segmentstore.segment.write_latency_ms
+
+  ```
+
+- Segment Store global and per-segment Read/Write Metrics ([Counters](https://micrometer.io/docs/concepts#_counters)):
+
+   ```
+      // Global counters
+      pravega_segmentstore.segment.read_bytes_global
+      pravega_segmentstore.segment.write_bytes_global
+      pravega_segmentstore.segment.write_events_global
+
+      // Per segment counters - all with tags {"scope", $scope, "stream", $stream, "segment", $segment, "epoch", $epoch}
+
+      pravega_segmentstore.segment.write_bytes
+      pravega_segmentstore.segment.read_bytes
+      pravega_segmentstore.segment.write_events
```

Comment:
@kevinhan88 added prefis pravega_..please check

Reply:
at least it should be "pravega.", but I prefer omitting the prefix, because it is configurable.

Reply:
ok